ios/RNKiwiMobile/RNKiwiGestureController.m

```diff
@@ -0,0 +1,22 @@
+
+#import <Foundation/Foundation.h>
+#import "RNKiwiGestureController.h"
+
+NSString *const RNKiwiAddGestures = @"RNKiwiAddGestures";
+NSString *const RNKiwiRemoveGestures = @"RNKiwiRemoveGestures";
```

Comment:
I like that :) As a general "good practice", I suggest adding one doc block that describes when this gets called (or should be)

Reply:
We can also mention it is scoped to `moduleName`